pyoes/templates/pyoes/header.jinja2

```diff
@@ -12,25 +12,27 @@
 
     {% set home_link = home_link|default('/') -%}
     <!-- logo -->
-    <div class="medium-8 small-12 columns">
+    <div class="medium-8 small-12 columns row">
```

Comment:
ik weet niet of ik ooit al colums & row op dezelfde div gezet heb, meestal worden die genest. maar if it works..

Reply:
Ja, het is echt een gepruts om dat op alle screen sizes goed te krijgen.